package.json

```diff
@@ -44,5 +44,6 @@
   "files": [
     "tasks",
     "LICENSE-MIT"
-  ]
+  ],
+  "appveyor_id": "fmp4q531iui753to"
```

Comment:
They changed it recently (1-2 months ago) so that YAML is strict. I can't find the post atm, but @FeodorFitsner had CC'ed me back then.